libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -404,3 +404,30 @@ export const firetextConfig: IConfigCredentials[] = [
   },
   ...smsConfigBase,
 ];
+
+export const bulksmsConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.AuthKey,
+    displayName: 'Auth Key',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Sender,
+    displayName: 'Sender',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Route,
+    displayName: 'Route',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.DltEntityId,
+    displayName: 'DLT Entity Id',
+    type: 'string',
+    required: false,
+  },
```

Comment:
As with the other SMS providers this needs to depend of `smsConfigBase`. That will also help @djabarovgeorge suggestion of using `from` rather than this new field added `sender`.

Reply:
@p-fernandez instead of sender I have added smsConfigBase you can check now.

Reply:
I am sorry but the change you mention is not showing up in the PR. Please review.